common/utils/src/main/resources/error/error-classes.json

```diff
@@ -916,6 +916,12 @@
       }
     }
   },
+  "EXPRESSION_DECODING_FAILED" : {
+    "message" : [
+      "Error while decoding: <e>",
```

Comment:
Let's not include the exception `<e>` to the error message because of:
1. The message is visible not only to Java/Scala users but also PySpark and SQL users
2. It is included as `cause` in `SparkRuntimeException`

Reply:
Good catch, I think the object of "Error while decoding" should be expression not the exception.